docs/visualize/timelion.asciidoc

```diff
@@ -504,3 +504,61 @@ image::images/timelion-conditional04.png[]
 {nbsp}
 
 For additional information on Timelion conditional capabilities, go to https://www.elastic.co/blog/timeseries-if-then-else-with-timelion[I have but one .condition()].
+
+[float]
+[[timelion-deprecation]]
+=== Timelion App deprecation
+
+In *8.0* and later, the Timelion app is going to be deprecated. There are actions you need to complete to copy your existing Timelion worksheets to a dashboard.  
```

Comment:
We remove it in 8.0 and it's deprecated since 7.0. I think we should be more clear here that 8.0 is the removal date.